battle.h

```diff
@@ -75,11 +78,13 @@ class Battle : public AListElem
 		//
 		void UpdateShields(Army *);
 		void DoSpecialAttack( int round, Soldier *a, Army *attackers,
-				Army *def, int behind );
+				Army *def, int behind, int canattackback);
 
 		void WriteSides(ARegion *,Unit *,Unit *,AList *,AList *,int,
 				ARegionList *pRegs );
 
+		// void WriteBattleStats(ArmyStats *);
```

Comment:
Need to investigate why it was commended

Reply:
Similarly, it looks like that was your initial attempt from 3 years ago to create some battle statistics and has been replaced with newer/better code (the BattleLogLevel::Detailed stuff in the normal battle output generation).